MVC/application/controller/afterLogin.php

```diff
@@ -46,8 +46,45 @@ public function userProfile() {
       else {
         session_destroy();
         echo "<script>alert('First log-in!!!');</script>";
-        //header("location: /userControl");
-        $this->view("userControl");
+        header("location: /userControl");
+        //$this->view("userControl");
+      }
+    }
+
+    public function postData() {
+      session_start();
+      if(isset($_SESSION['logged_in']) && $_SESSION['logged_in']) {
```

Comment:
We should check whether the user is logged in or not at the time of loading the welcome page. So there's no need of checking whether the user is logged in or not at the time of posting, because we can redirect the user to the login page at the previous step i.e loading the welcome page

Reply:
Yes we can but here, I used for if any user directly try to access this metjos using `url` then it will check once again where user is logged in or not.